docs/eventlisteners.md

```diff
@@ -220,6 +221,24 @@ must adhere to the
 [Kubernetes syntax and character set requirements](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
 for label values.
 
+## Annotations
```

Comment:
I think we should also document the behaviour for what happens when deployment or service has an annotation already present.

Reply:
Or is mentioning that only in release note OK, @dibyom ?

Reply:
@khrm I have updated the doc also with the `Note`